engine.go

```diff
@@ -243,12 +243,12 @@ func (e *RTCEngine) waitUntilConnected() error {
 	}
 }
 
-func (e *RTCEngine) ensurePublisherConnected() error {
+func (e *RTCEngine) ensurePublisherConnected(dataReady bool) error {
```

Comment:
Maybe rename this to `checkDataReady`. Just `dataReady` feels like it is a signal about readiness of data channel. Or maybe `ignoreDataReady`.

Reply:
done